packages/theme-default/src/components/index.tsx

```diff
@@ -14,7 +14,7 @@ export { PackageManagerTabs } from './PackageManagerTabs';
 export { PrevNextPage } from './PrevNextPage';
 export { ScrollToTop } from './ScrollToTop';
 export { Search, SearchPanel } from './Search';
-export { SideBar } from './SideBar';
+export { SideBar } from './Sidebar';
```

Comment:
the component name is SideBar, should the file name be the same with the component name?

Reply:
I think it's better to use the same name and i have updated it